src/main/java/com/hivemq/cli/graal/NativeMain.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2019-present HiveMQ and the HiveMQ Community
```

Comment:
Can this class be removed?

Reply:
This class is needed for reflection config generation.